openlibrary/plugins/upstream/account.py

```diff
@@ -56,7 +55,7 @@
 RESULTS_PER_PAGE: Final = 25
 
 # XXX: These need to be cleaned up
-send_verification_email = accounts.send_verification_email
+# send_verification_email = accounts.send_verification_email
```

Comment:
Should this be deleted?

Reply:
I think so.  I don't see it referenced anywhere.